docs/templates/pagerduty-integration.md

```diff
@@ -76,7 +76,7 @@ If you need help with this integration, please contact ***[support@fiberplane.co
 1. Go to the **Incidents** screen in PagerDuty and click **+ New Incident**
 2. Select an **Impacted Service** and add a **Title** to the incident and optionally select an **Urgency, Incident Priority** and **Description.**
 3. Click **Create Incident**
-4. Click **! Acknowledge**
+4. Click **Acknowledge**
```

Comment:
The button on the Pagerduty interface that this line refers to has an exclamation mark so this change is not necessary:

```suggestion
4. Click **! Acknowledge**
```

Reply:
Done!